cal/api/urls.py

```diff
@@ -15,12 +15,12 @@
     url(r'^v1/colorcategories/$', views.ColorCategoryList.as_view()),
     url(r'^v1/colorcategories/(?P<pk>[0-9]+)/?$', views.ColorCategoryDetail.as_view()),
     url(r'^v1/colorcategories/(?P<pk>[0-9]+)/events/?$', views.ColorCategoryDetailEvents.as_view()),
-    url(r'^v1/colorcategories/(?P<pk>[0-9]+)/eventWeek/?$', views.ColorCategoryDetailEventTimeSeries.as_view()),
+    url(r'^v1/colorcategories/(?P<pk>[0-9]+)/event/(?P<time_step>\w+)/?$', views.ColorCategoryDetailEventTimeSeries.as_view()),
```

Comment:
Sorry to make you change this again, but I don't think `/event/` is the best RESTful structure, since we're not returning events for a specific category, we're returning a time series for a specific category. What about we change it to `/v1/colorcategories/<pk>/timeseries/<time_step>/`